### PR TITLE
Segment Reducer Issue

### PR DESCRIPTION
**Pull request by dadamsretrorabbit**
Reducer Removed in Merge Conflicts

### CODE REVIEW

File: waterlog/src/App.scss [hunk 1]
```diff
@@ -2,216 +2,189 @@
...
-  margin-top: 20px;
+	margin-top: 20px;
 }
 
 .bold {
-  font-weight: bold;
+	font-weight: bold;
 }
 
 .wastageInfo {
```
Comment (FredJEhlers) [inline waterlog/src/App.scss]: inconsistent naming scheme `.btn-graph-nav` vs `.wastageInfo` your team needs to choose one and use it in all css files

File: waterlog/src/Components/Tank.js [hunk 2]
```diff
@@ -1,50 +1,15 @@
...
-
+	const { id, percentageLevel } = props.tank;
   return (
-    <div className="tank">
-      <p>Tank {props.tank.id}</p>
-      {getTankImage(props.tank.percentageLevel)}
+		<div className="tank" onClick={() => (window.location = `/alert/tank/${id}`)}>
+			<p>Tank {props.tank.id}</p>
+			<img src={getTankImage(percentageLevel)}
+				className={(percentageLevel === 0 || percentageLevel > 80) ? 'exlamation': ''}
```
Comment (FredJEhlers) [inline waterlog/src/Components/Tank.js]: spelling mistake `exlamation` -> `exclamation

File: waterlog/src/reducers/PMapsReducer.js [hunk 3]
```diff
@@ -1,39 +1,35 @@
-import {
-    FETCH_PMAP_DATA_BEGIN,
-    FETCH_PMAP_DATA_SUCCESS,
-    FETCH_PMAP_DATA_FAILURE
-} from '../actions/Types';
+import { FETCH_PMAP_DATA_BEGIN, FETCH_PMAP_DATA_SUCCESS, FETCH_PMAP_DATA_FAILURE } from '../actions/Types';
```
Comment (FredJEhlers) [inline waterlog/src/reducers/PMapsReducer.js]: This was correct, Please change it back to one import per line

File: waterlog/src/utils.js [hunk 4]
```diff
@@ -1,179 +1,167 @@
...
-  Marker,
-  Popup
-} from "react-leaflet";
-import {
-  sensorFaultIcon,
-  sensorOkLightIcon,
-  sensorOkDarkerIcon
-} from "./icons/MapIcons";
+import React from 'react';
+import { Polyline, Marker, Popup } from 'react-leaflet';
```
Comment (FredJEhlers) [inline waterlog/src/utils.js]: Each import on each line like before please

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/Tank.js (hunk 2 above)
Comment (cmanamela) [reply waterlog/src/Components/Tank.js]: i was wondering why it was not working

File: waterlog/src/reducers/PMapsReducer.js (hunk 3 above)
Comment (cmanamela) [reply waterlog/src/reducers/PMapsReducer.js]: Fixed

File: waterlog/src/utils.js (hunk 4 above)
Comment (cmanamela) [reply waterlog/src/utils.js]: Done

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**